src/main/webapp/components/Navbar.vue

```diff
@@ -1,58 +1,67 @@
 <template>
-  <nav class="navy-blue sticky nav-bar">
-    <a href="index.html" class="bold" id="title">starfish</a>
-    <img src="/assets/starfish.png" id="logo">
-    <div class="action-items" v-if="!signedIn">
+  <div style="height:80px">
```

Comment:
Prefer using class styling and scss over inline styling. Better to have it grouped in one spot

Reply:
Ok I'll change that